packages/api/test/dist.test.ts

```diff
@@ -34,8 +34,8 @@ describe('typescript dist verification', function () {
   });
 
   it('should be able to set an auth token', async function () {
-    const user = 'username';
-    const pass = 'changeme';
+    const user = 'buster';
```

Comment:
Changed these from `username` everywhere so it's clearer that we have a real username in use (and not somehow turning a `username` variable into a string literal).

<sub>also :buster-chomp:</sub>